src/Libraries/Microsoft.Extensions.AI.Abstractions/ChatCompletion/ChatModelMetadata.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.Extensions.AI;
+
+/// <summary>Provides metadata about a model used with an <see cref="IChatClient"/>.</summary>
+public class ChatModelMetadata
+{
+    /// <summary>
+    /// Gets a value indicating whether the model can produce structured output conforming to a JSON schema.
+    /// </summary>
+    public bool? SupportsNativeJsonSchema { get; init; }
```

Comment:
Not sure about the naming any more. Is the word "native" doing anything useful here?

Could be "SupportsStrictJsonSchema" or just "SupportsJsonSchema".

Could be "SupportsStructuredOutput" but the only kind we support is JSON schema.

Reply:
I'd drop "native", both here and in the parameter name to `GetResponseAsync<T>`. SupportsJsonSchema, SupportsJsonSchemaResponseFormat, and SupportsStructuredOutput all sound fine to me.

Reply:
Sounds good.

Since the metadata property isn't something normal app code would use, I've gone with the most explicit phrasing (`SupportsJsonSchemaResponseFormat`), just in case other kinds of structured output emerge in the future.

For the `GetResponseAsync<T>` parameter, it's now `useJsonSchema`.